mkdocs_awesome_pages_plugin/meta.py

```diff
@@ -34,8 +34,11 @@ def from_yaml(item: Union[str, dict], context: str):
 
         if isinstance(item, dict) and len(item) == 1:
             (title, value) = list(item.items())[0]
-            if isinstance(value, str) and isinstance(title, str):
-                return MetaNavItem(value, title)
+            if isinstance(title, str):
+                if isinstance(value, str):
+                    return MetaNavItem(value, title)
+                elif isinstance(value, list):
+                    return MetaNavItem([MetaNavItem.from_yaml(it, context) for it in value], title)
```

Comment:
The type annotation of `value` needs to be extended for this. `Union[str, List[MetaNavItem]]` should do the trick.